src/main/java/com/kodilla/ecommercee/exceptions/CartNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.kodilla.ecommercee.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.NOT_FOUND,reason = "Cart not found")
```

Comment:
Cart not found > Cart not found!
by zachowac ciaglosc komunikatow ;)

Reply:
Oh...no tak :)